bip-XXXX.mediawiki

```diff
@@ -0,0 +1,138 @@
+<pre>
+  BIP: ???
+  Layer: Applications
+  Title: Signatures of Messages using Bitcoin Private Keys
+  Author: Christopher Gilliard &ltchristopher.gilliard@gmail.com&gt
+  Comments-Summary: No comments yet.
+  Comments-URI: TBD
+  Status: Proposal
+  Type: Informational
+  Created: 2019-02-16
+  License: BSD-2-Clause
+</pre>
+
+# Abstract #
+
+This document describes a signature format for signing messages with Bitcoin private keys.
+
+The specification is intended to describe the standard for signatures of messages that can be signed and verfied between different clients that exist in the field today. Note: that a new signature format has been definied which has a number of advantages over this BIP, but to be backwards compatible with existing implementations this BIP will be useful. See BIP 322 [1] for full details on the new signature scheme.
+
+One of the key problems in this area is that there are several different types of Bitcoin addresses and without introducing specific standards it is unclear which type of address format is being used. See [2]. This BIP will attempt to address these issues and define a clear and concise format for Bitcoin signatures.
+
+# Copyright #
+
+This BIP is licensed under the 2-clause BSD license.
+
+# Motivation #
+
+Since Bitcoin private keys can not only be used to sign Bitcoin transactions, but also any other message, it has become customary to use them to sign various messages for differing purposes. Some applications of signing messages with a Bitcoin private key are as follows: proof of funds for collateral, credit worthiness, enterence to events, airdrops, audits as well as other applications. While there was no BIP written for how to digitally sign messages with Bitcoin private keys with P2PKH addresses it is a fairly well understood process, however with the introduction of Segwit (both in the form of P2SH and bech32) addresses, it is unclear how to distinguish a P2PKH, P2SH, or bech32 address from one another. This BIP proposes a standard signature format that will allow clients to distinguish between the different address formats.
+
+# Specification #
+
+## Background on ECDSA Signatures ##
+
+(For readers who already understand how ECDSA signatures work, you can skip this section as this is only intended as background information.)
+Elliptic Curve Digital Signature Algorithm or ECDSA is a cryptographic algorithm used by Bitcoin to ensure that funds can only be spent by their rightful owners.
+
+A few concepts related to ECDSA:
+
+private key: A secret number, known only to the person that generated it. A private key is essentially a randomly generated number. In Bitcoin, someone with the private key that corresponds to funds on the block chain can spend the funds. In Bitcoin, a private key is a single unsigned 256 bit integer (32 bytes).
```

Comment:
Need some sort of sub-header or bold font for these terms. In the output, these are jumbled together into the paragraph, making it hard to read. Maybe just blank lines and bold formatting for "private key," "public key," and "signature."

Reply:
Fixed here: https://github.com/bitcoin/bips/pull/764/commits/99f4211dde9f0b3ae31f366303c225e314638f1c